pkg/local_workflows/auth_workflow.go

```diff
@@ -71,34 +68,42 @@ func authEntryPoint(invocationCtx workflow.InvocationContext, _ []workflow.Data)
 		auth.WithLogger(logger),
 	)
 
-	err = entryPointDI(config, logger, engine, authenticator)
+	err = entryPointDI(invocationCtx, logger, engine, authenticator)
 	return nil, err
 }
 
 func autoDetectAuthType(config configuration.Configuration) string {
 	// testing if an API token was specified
 	token := config.GetString(ConfigurationNewAuthenticationToken)
-	if _, uuidErr := uuid.Parse(token); uuidErr == nil {
-		return authTypeToken
+	if len(token) > 0 && auth.IsAuthTypeToken(token) {
+		return auth.AUTH_TYPE_TOKEN
 	}
 
+	// currently the auth workflow defaults to traditional token auth when an IDE environment is detected
+	// this will need to change if IDEs want to invoke this workflow for oauth/PAT
 	integration := config.GetString(configuration.INTEGRATION_NAME)
 	if pgk_utils.IsSnykIde(integration) {
-		return authTypeToken
+		return auth.AUTH_TYPE_TOKEN
```

Comment:
this should just be removed

Reply:
I think it is there for older IDEs, that do not explicitly set the auth type.